app/views/main/home.html.erb

```diff
@@ -32,11 +32,6 @@
     </div>
   </div>
 </div><!-- end yield -->
-<div class="github-fork-ribbon-wrapper right-bottom">
-  <div class="github-fork-ribbon">
-    <a href="https://github.com/metamaps/metamaps" target="_blank">Fork me on GitHub</a>
-  </div>
-</div>           
```

Comment:
No, it's because in my new homepage mockup, I am thinking we can add a dedicated little section to the code itself.